### ISSUE TITLE
Updating Dynamo Packages (pkg.json)

### ISSUE THREAD

**Issue by moethu** [DynamoDS/Dynamo#8319]
When updating packages the prescribed way (http://dynamoprimer.com/en/10_Packages/10-4_Publishing.html) pkg.json doesn't always end up for me like it should be.

## Dynamo version
1.3

## Operating system
Win7 64

## What did you do? 
Updated an existing package

## What did you expect to see?
Updated package with pkg.json correctly updated - key assemblies (node_libraries) should not be changed.

## What did you see instead?
pkg.json has been modified so my key assemblies are no longer listed:
```
"node_libraries":["KonstruDynamo.customization, Version=1.6.4.0, Culture=neutral, PublicKeyToken=null","KonstruDynamoUI.customization, Version=1.6.4.0, Culture=neutral, PublicKeyToken=null"]}
```
Only listing customization dlls but no node libraries.

**Comment by mjkkirschner** [comment]
@moethu I think this happens when you increment your dll version - was that the case?

**Comment by moethu** [comment]
yes, this would make sense. if so, I would request to change it so node libraries are identified by name only and therefore remain sticky. One mostly updates packages to provide bugfixes & features with incremented dll versions.

**Comment by andydandy74** [comment]
I‘d second that request. This has happened to me before as well.
